### ISSUE TITLE
Expansões

### ISSUE THREAD

**Issue by anacastrosalgado** [anacastrosalgado/DACL#20]
Expandir:

ing. inglês (etimologia)
gr. grego (etimologia)
lat. latim (etimologia)
Do part. pas. do v. Do particípio passado do verbo (etimologia)
suf. sufixo (sufixo)
it. italiano

**Comment by ambs** [comment]
Isto é tudo etimologia, certo?
Acho que ainda não me dediquei a expandir nada na etimologia.

**Comment by anacastrosalgado** [comment]
Sim, é. Se possível, era necessário expandir todas as abreviaturas em etimologia. Poupava-me imenso trabalho.

**Comment by ambs** [comment]
Queres que as expanda no documento original, ou seja, diretamente no XML, ou queres que faça como as outras?
Na verdade, para estas, acho que preferia expandir diretamente no XML. Pode ser?

**Comment by anacastrosalgado** [comment]
Sim, podes expandir no xml. É o que estou a fazer manualmente, na verdade.

**Comment by ambs** [comment]
OK, faço isso hoje à noite :)

Alberto Simões

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by anacastrosalgado** [comment]
Ora bem, isto é de 21 de setembro de 2020, Sua Excelência recorda-se de o ter feito? :)
